tensorboard/webapp/notification_center/_data_source/notification_center_data_source.ts

```diff
@@ -32,4 +33,15 @@ export class TBNotificationCenterDataSource
   fetchNotifications(): Observable<NotificationCenterResponse> {
     return this.http.get<NotificationCenterResponse>(`data/notifications`);
   }
+
+  updateAndGetLastReadTimeStampInMs(): Observable<number> {
+    const lasReadTime =
+      window.localStorage.getItem(NOTIFICATION_LAST_READ_TIME_KEY) ?? '-1';
+    const timeNow = Date.now();
+    window.localStorage.setItem(
+      NOTIFICATION_LAST_READ_TIME_KEY,
+      timeNow.toString()
+    );
+    return of(parseInt(lasReadTime));
```

Comment:
Internal lint tools would complain about this. Use `Number(lastReadTime)` or `parseInt(lastReadTime, 10)` (specify the redix)